index.html

```diff
@@ -6,6 +6,9 @@
     <title>repl.it</title>
     <link href="style.css" rel="stylesheet" type="text/css" />
     <script src="https://cdn.jsdelivr.net/npm/p5@1.3.1/lib/p5.js"></script>
+
+    <script src="path/to/p5.sound.js"></script>
```

Comment:
this needs to be a link to a p5.sound file! (like above)

you could use this: `<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.3.1/addons/p5.sound.js"></script>`

Reply:
right now your sound doesn't work (you probably noticed :p)